src/sparkline.c

```diff
@@ -95,7 +95,7 @@ void freeSparklineSequence(struct sequence *seq) {
  * ASCII rendering of sequence
  * ------------------------------------------------------------------------- */
 
-/* Render part of a sequence, so that render_sequence() call call this function
+/* Render part of a sequence, so that render_sequence() calls this function
```

Comment:
atypical correction. I'm not sure about all of the aspects. It's possible it should be `call calls` ...